src/ic3/m1ain.cc

```diff
@@ -135,7 +135,7 @@ int ic3_enginet::operator()()
     {
       namespacet ns(transition_system.symbol_table);
       report_results(cmdline, properties, ns, message.get_message_handler());
-      return 10;
+      return properties.exit_code();
```

Comment:
I’m still confused by number-of-properties-equals-zero - should that have been not-equal-to-zero?

Reply:
No; the IC3 code right now doesn't use the standard reporting. Will fix.